src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -31,6 +34,13 @@ class UserManagementFunctionalTest extends FunctionalTest {
     private static final String EMAIL_ADDRESS = "Email-Address";
     private static final String COURTHOUSE_ID = "courthouse_id";
     private static final String ADMIN_USERS = "/admin/users";
+    static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
We should probably consider adding the `ObjectMapperConfig` class to the `FunctionalTest` SpringBootTest to allow us to autowire the already configured ObjectMapper we have.